### PR TITLE
feat: cmp traits for json_types::integers (#822)

### PR DESCRIPTION
**Pull request by curryrasul**

### CODE REVIEW

File: near-sdk/src/json_types/integers.rs [hunk 1]
```diff
@@ -80,6 +82,7 @@ mod tests {
         test_serde!(U128, u128, 10u128.pow(18));
         test_serde!(U128, u128, 2u128.pow(100));
         test_serde!(U128, u128, u128::max_value());
+        assert!(U128::from(u128::min_value()) < U128::from(u128::max_value()));
```
Comment (austinabell) [inline near-sdk/src/json_types/integers.rs:85]: I don't think these tests are really necessary for things like this, and only test `PartialOrd`, but I think it's fine to keep!

**Review by austinabell — APPROVED** [review]

**Comment by curryrasul** [comment]
Should I do something else on this PR ?

**Comment by willemneal** [comment]
@curryrasul It looks like you are out of date with the base branch so a rebase or merge is needed.

Then you'll need to install Docker and run the build_all.sh script in the examples directory to rebuild all of the example binaries.  I think this requirement is a bit much and bloats the git history with wasm blobs, but as you can see form this lenik it's needed to pass this CI check: https://buildkite.com/nearprotocol/near-sdk-rs/builds/4085#84cdfb22-4179-4247-93d8-42174f6dec5f/114-2779

Lastly someone with the power (not me) needs to approve your PR to be tested by Github Actions.  

Thanks for the quick PR!

**Comment by austinabell** [comment]
> Should I do something else on this PR ? I don't have experience contributing to a big open source, so I don't know how the process is going (even after reading CoC and contributing.md) 

Nope, sorry this PR just had to wait for a few others to come in first that were time sensitive. I'll pull it in soon, nothing to do on your end

**Pull request merged**